recipes/recipes_emscripten/lz4-c/recipe.yaml

```diff
@@ -0,0 +1,52 @@
+context:
+  pkg_name: lz4-c
+  name: lz4
+  version: 1.10.0
+
+package:
+  name: ${{ pkg_name }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/${{ name }}/${{ name }}/archive/v${{ version }}.tar.gz
+  sha256: 537512904744b35e232912055ccf8ec66d768639ff3abe5788d90d792ec5f48b
+  #patches:
+  #- patches/CVE-2021-3520.patch
```

Comment:
It looks like in the original feedstock the patches are not applied, I guess I should just remove this.